sampleapp/src/main/AndroidManifest.xml

```diff
@@ -17,7 +18,8 @@
             </intent-filter>
         </activity>
 
-        <service android:name="io.outbound.sdk.OutboundMessagingService">
+        <!--<service android:name="io.outbound.sdk.OutboundMessagingService">-->
```

Comment:
Is there a reason to keep the commented line? I'd rather see it removed.

Reply:
Integrators have a choice between using the `OutboundMessagingService` or subclassing their own version and registering that instead. I kept the commented version here to demonstrate both ways, but it's not really necessary so I'll remove it.